clients/AlphabotClientLibrary/AlphabotClientLibrary.Shared/Responses/MagnetometerResponse.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using AlphabotClientLibrary.Shared.Contracts;
+using AlphabotClientLibrary.Shared.Models;
+
+namespace AlphabotClientLibrary.Shared.Responses
+{
+    public class MagnetometerResponse : IAlphabotResponse
+    {
+        /// <summary>
+        /// The magnetic flux density in µT
```

Comment:
Do not use non-ASCII characters
```suggestion
        /// The magnetic flux density in Microtesla
```